_posts/2018-04-11-dont-make-users-think-about-updating-software.md

```diff
@@ -11,35 +11,35 @@ tags:
   - update-service
 ---
 
-As software developers, we want the people using the software we create to ~~enjoy~~ not _hate_ the experience. Developers strive to create a positive user experience to increase the popularity and usage of their software. Consequently, many hours are spent debating and designing user interactions that are simple, effective, and intuitive.
+As software developers, we want the people who are using our software to ~~enjoy~~ _not hate_ the experience. Developers strive to create a positive user experience to increase the popularity and usage of their software. Consequently, we spend many hours designing, debating, and redesigning to ensure that user interactions with our software are simple, effective, and intuitive.
```

Comment:
These are mostly just changes for readability. If you prefer the original, feel free to change it back.